SPECS/mysql/mysql.spec

```diff
@@ -75,9 +76,12 @@ make test
 %{_libdir}/pkgconfig/mysqlclient.pc
 
 %changelog
-* Tue Nov 03 2020 Rachel Menge <rachelmenge@microsoft.com> - 8.0.22-1
-- Upgrade to 8.0.22. Fixes 40 CVES.
-- Lint spec
+* Thu Nov 05 2020 Rachel Menge <rachelmenge@microsoft.com> - 8.0.22-2
+- Added no patch for CVE-2012-5627
```

Comment:
Nit: could you unify the alignment in the changelog? Either one way or another as long as it's uniform.

Reply:
This is a result of the spec cleaner / linter. @thcrain-msft is this still the preferred format?
![image](https://user-images.githubusercontent.com/10325590/98392356-e2730680-2025-11eb-9ade-af4928b5e9b5.png)

Reply:
Yep! Preferred style is one space between */- and the rest of the line, one blank line between entries, and a dash separating the email from the version-release string.